streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -418,6 +422,11 @@
                     0,
                     Importance.MEDIUM,
                     NUM_STANDBY_REPLICAS_DOC)
+            .define(RACK_ID_CONFIG,
```

Comment:
Nit: we sort config based on `Importance` and within each importance group alphabetically. Please move the config to the correct place.